src/server/gc_worker/compaction_filter.rs

```diff
@@ -62,9 +62,10 @@ lazy_static! {
     )
     .unwrap();
     // A counter for errors met by `WriteCompactionFilter`.
-    static ref GC_COMPACTION_FAILURE: IntCounter = register_int_counter!(
+    static ref GC_COMPACTION_FAILURE: IntCounterVec = register_int_counter_vec!(
```

Comment:
Does the grafana dashboard need to be changed?

Reply:
Is it safe to add a dimension to the existing metrics?

Reply:
"Is it safe to add a dimension to the existing metrics?" changed it in my test cluster, doesn't encounter any problem.